src/yunohost/app.py

```diff
@@ -1219,14 +1186,25 @@ def app_ssowatconf():
 
     main_domain = _get_maindomain()
     domains = domain_list()['domains']
-    all_permissions = user_permission_list(full=True)['permissions']
-
-    skipped_urls = []
-    skipped_regex = []
-    unprotected_urls = []
-    unprotected_regex = []
-    protected_urls = []
-    protected_regex = []
+    all_permissions = user_permission_list(full=True, ignore_system_perms=True, full_path=True)['permissions']
+
+    permissions = {
+        'core_skipped': {
+            "users": [],
+            "label": "Core permissions - skipped",
+            "show_tile": False,
+            "auth_header": False,
+            "public": False,
+            "uris": [
+                [domain + '/yunohost/admin' for domain in domains] + \
+                [domain + '/yunohost/api' for domain in domains] + [
+                    "re:^[^/]*/%.well%-known/ynh%-diagnosis/.*$",
+                    "re:^[^/]*/%.well%-known/acme%-challenge/.*$",
+                    "re:^[^/]*/%.well%-known/autoconfig/mail/config%-v1%.1%.xml.*$"
+                ]
```

Comment:
There is a double array here
On my `ynh-dev`:
```
"uris": [
                [
                    "domain.tld/yunohost/admin", 
                    "domain.tld/yunohost/api", 
                    "re:^[^/]*/%.well%-known/ynh%-diagnosis/.*$", 
                    "re:^[^/]*/%.well%-known/acme%-challenge/.*$", 
                    "re:^[^/]*/%.well%-known/autoconfig/mail/config%-v1%.1%.xml.*$"
                ]
]
```

Reply:
Thanks, maybe it's way I have some issue with ssowat. Should be fixed now